src/size.cpp

```diff
@@ -133,18 +135,24 @@ double obj_size_tree(SEXP x,
   // Linked lists
   case DOTSXP:
   case LISTSXP:
-  case LANGSXP:
-    if (x == R_MissingArg) // Needed for DOTSXP
+  case LANGSXP: {
```

Comment:
Does this change mean that we can drop the special casing for deferred string ALTREPs? (line 72 above)

Reply:
Indeed! The special-casing is now removed.